src/test/java/api/loans/DueDateScheduledNoticesProcessingTests.java

```diff
@@ -419,6 +437,44 @@ void testNoticesForNonExistentLoansDoNotBlockTheQueue() {
     verifyNumberOfPublishedEvents(NOTICE_ERROR, 2);
   }
 
+  @Test
+  void testNoticesForNonExistentPatronGroupsDoNotBlockTheQueue() {
+    generateLoanAndScheduledNotices();
+
+    int expectedNumberOfUnprocessedNotices = 0;
+
+    List<JsonObject> notices = createNoticesOverTime(dueDate.minusMinutes(1)::minusHours, 4);
+
+    var basedUponNod = itemsFixture.basedUponNod();
+    var jessica = usersFixture.jessica();
+
+    var jessicaNodLoan = checkOutFixture.checkOutByBarcode(basedUponNod, jessica);
+
+    usersClient.replace(borrower.getId(), new UserBuilder().withPatronGroupId(null));
```

Comment:
Again, this is not what the test name implies. Please rename the test OR delete the patron group OR change `patronGroupId` to a non-existent one.

Reply:
Test renamed